generics/constants/common.js

```diff
@@ -150,5 +150,9 @@ module.exports = {
 	DELETE_EVENT_ENTITY: 'user',
 	DELETE_EVENT_TYPE: 'delete',
 	OBJECT: 'object',
+	STRING: 'string',
+	ALL: 'all',
```

Comment:
@praveenKDass , can we reuse ALL_SCOPE_VALUE instead of using "ALL" directly wherever it's being used?